### DISCUSSION TITLE
multiple pk error nocodb [May 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2264] [Discord Threads]
> 🙋‍♂️ **Canx**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Canx**  
#### Still haven't found solution for this... Somehow seems that nocodb does not detect that table has composite primary key. Just pasting SQL table definition if someone has any hint: "CREATE TABLE IF NOT EXISTS public.nc_u10l__sesiones   
#### (   
####     created_at timestamp without time zone DEFAULT now(),   
####     updated_at timestamp without time zone DEFAULT now(),   
####     fecha character varying COLLATE pg_catalog."default",   
####     "diaSemana" character varying COLLATE pg_catalog."default",   
####     "horaDesde" character varying COLLATE pg_catalog."default",   
####     "horaHasta" character varying COLLATE pg_catalog."default",   
####     "asignaturaNombre" character varying COLLATE pg_catalog."default",   
####     nombre character varying COLLATE pg_catalog."default",   
####     curso character varying COLLATE pg_catalog."default",   
####     car character varying COLLATE pg_catalog."default",   
####     "sessionId" bigint NOT NULL,   
####     "horarioId" bigint NOT NULL,   
####     CONSTRAINT nc_u10l__sesiones_pkey PRIMARY KEY ("sessionId", "horarioId")   
#### )"  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/977863816782155857)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
